src/game.cpp

```diff
@@ -6202,7 +6202,7 @@ void game::print_all_tile_info( const tripoint &lp, const catacurses::window &w_
     auto this_sound = sounds::sound_at( lp );
     if( !this_sound.empty() ) {
         const int lines = fold_and_print( w_look, point( 1, ++line ), max_width, c_light_gray,
-                                          _( "You heard %s from here." ),
+                                          _( "From here you heard %s" ),
```

Comment:
```suggestion
                                          _( "From here you heard: %s" ),
```
Admittedly a bit off when the sound is "footsteps", but much better for "whump".

Reply:
I'm not convinced a colon is better here. The existing message log does not include one, so I would like to keep them consistent.

Reply:
I'm not sure why changing the words and adding a colon are that different.

Reply:
What I mean is, I intended to make these consistent with the existing messages in the sidebar log, as shown in the screenshots above. `From the the south and below you heard whump.` is logged, and `From here you heard whump.` is indicated when you examine the sound. Adding a colon to one (but not the other) would make them less consistent with one another, for no clear benefit.